<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProductLevelSettingWidget</class>
 <widget class="QWidget" name="ProductLevelSettingWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>651</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ProductLevelSettingWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,1,0,2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,0,1">
     <item>
      <widget class="QPushButton" name="btn_operator">
       <property name="text">
        <string>操作</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="combo_roi"/>
     </item>
     <item>
      <widget class="QPushButton" name="btn_roi">
       <property name="text">
        <string>划定roi</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btn_roi_color">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_roi">
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="CheckSurfaceRoiWidget" name="roi_widget" native="true"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="button_add_defect">
       <property name="text">
        <string>添加缺陷</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_logic">
       <property name="text">
        <string>逻辑运算符</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_expression">
       <property name="text">
        <string>添加条件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_delete">
       <property name="text">
        <string>删除条件</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_clear">
       <property name="text">
        <string>清  空</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTreeWidget" name="tree_rule">
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectItems</enum>
     </property>
     <property name="headerHidden">
      <bool>true</bool>
     </property>
     <property name="columnCount">
      <number>2</number>
     </property>
     <column>
      <property name="text">
       <string notr="true">1</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string notr="true">2</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CheckSurfaceRoiWidget</class>
   <extends>QWidget</extends>
   <header>checksurfaceroiwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
